guides/common/modules/proc_managing-and-monitoring-hosts-using-cockpit.adoc

```diff
@@ -5,14 +5,15 @@ You can access the {Cockpit} UI through the {ProjectWebUI} and use the functiona
 
 .Prerequisites
 * You have enabled {the-Cockpit} in {Project}.
-* You have installed {the-Cockpit} on the host that you want to view:
-ifndef::satellite,orcharhino[]
-** For more information, see https://cockpit-project.org/running.html[Running Cockpit].
-endif::[]
-ifdef::satellite[]
-** For {RHEL}{nbsp}8, see {RHELDocsBaseURL}8/html/managing_systems_using_the_rhel_8_web_console/getting-started-with-the-rhel-8-web-console_system-management-using-the-rhel-8-web-console#installing-the-web-console_getting-started-with-the-rhel-8-web-console[Installing the web console] in _Managing systems using the RHEL{nbsp}8 web console_.
-** For {RHEL}{nbsp}7, see {RHELDocsBaseURL}7/html/managing_systems_using_the_rhel_7_web_console/getting-started-with-the-rhel-web-console_system-management-using-the-rhel-7-web-console#installing-the-web-console_getting-started-with-the-web-console[Installing the web console] in _Managing systems using the RHEL{nbsp}7 web console_.
-endif::[]
+* You have installed {the-Cockpit} on the host that you want to view.
++
+[NOTE]
+====
+You can use a job template to install {the-Cockpit}.
+To view the template in {ProjectWebUI}, navigate to *Monitor* > *Jobs* and click *Run Job*.
+From the *Job category* list, select *Ansible Services*.
+From the *Job template* list, select *Service Action {endash} Enable Web Console*.
+====
```

Comment:
I think this might be too much for a prerequisite but then again, I can't think of a better way to point users to the template. Does anyone have suggestions?

Reply:
I propose to move these steps to a procedure to which we can link.

Reply:
I'm not trying to describe a procedure, just the place where users can find the template. Do you have an alternative solution? What I'm trying to do is let users know that this template exists. I can't think of a way to do that without guiding them through the web UI.

Reply:
The procedure itself would be [13. Configuring and setting up remote jobs](https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#Configuring_and_Setting_Up_Remote_Jobs_managing-hosts). But how to best point to the particular template to use?

Reply:
I see this as a new 9.2 "Enabling Cockput on hosts": https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#host-management-and-monitoring-by-using-cockpit or similar.

The procedure itself does not need to duplicate all steps for REX jobs, but reference a procedure and mention the correct job category+job template.

Reply:
I don't know, I'd still like to avoid adding a whole new procedure. I wouldn't want to set a precedent for creating a whole new module every time I'd like to reference a job template/ansible module/etc. I still hope there is a simpler way to achieve that, one that doesn't eventually contribute to doc bloat :)

How about now? Just a note that the template is there and which category it's included in?

Reply:
Yes, I like this much better.